kodi_addon_checker/check_addon.py

```diff
@@ -1,20 +1,19 @@
 import os
-import xml.etree.ElementTree as ET
 import requests
 import logging
-from kodi_addon_checker import logger
 import gzip
 from io import BytesIO
 
-from kodi_addon_checker.record import Record, INFORMATION
+import xml.etree.ElementTree as ET
 from kodi_addon_checker.report import Report
+from kodi_addon_checker.record import Record, INFORMATION
 from kodi_addon_checker import check_artwork
 from kodi_addon_checker import check_old_addon
 from kodi_addon_checker import check_dependencies
 from kodi_addon_checker import check_entrypoint
-from kodi_addon_checker import handle_files
 from kodi_addon_checker import check_files
 from kodi_addon_checker import check_string
+from kodi_addon_checker import handle_files
```

Comment:
reordering has to be squashed into the commit which adds handle_files

Reply:
how do i squash only one line change into an older commit? i've only used squash to merge two entire commits.